src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -236,7 +278,7 @@ module.exports = function<T, P, I>(config : HostConfig<T, P, I>) : Reconciler {
     },
 
     unmountContainer(container : OpaqueNode) : void {
-      container.pendingProps = null;
+      container.pendingProps = [];
```

Comment:
The props here are the "children" of the container, however, `null` means "nothing to do" so we need a different way to render an empty set of children. :)

Not actually tested yet. Could also wrap it in an object like `{ children: null }` but seems unnecessary.